notification/message.go

```diff
@@ -22,3 +27,31 @@ const (
 	MessageTypeAlertBundle
 	MessageTypeAlertStatusBundle
 )
+
+func (s MessageType) Value() (driver.Value, error) {
+	switch s {
+	case 0:
+		return "alert_notification", nil
+	case 1:
+		return "alert_status_update", nil
+	case 2:
+		return "test_notification", nil
+	case 3:
+		return "verification_message", nil
+	case 4:
+		return "alert_notification_bundle", nil
+	case 5:
+		return "alert_status_update_bundle", nil
```

Comment:
Use the message type values (e.g. `MessageTypeAlertBundle` instead of `5`), not the `int` values.